src/components/SettingsPage/SettingNavigation.js

```diff
@@ -0,0 +1,46 @@
+import React, { Fragment } from "react";
+import { Link } from "react-router-dom";
+import { FormattedMessage } from "react-intl";
+import DashboardNavigationStyle from "../DashboardNavigation/DashboardNavigationStyle";
+
+export const SettingNavigation = () => {
+  const isCAreaPage = window.location.pathname.includes("area");
+  const isUsersPage = window.location.pathname.includes("users");
```

Comment:
Same here with regards to [NavLink](https://reacttraining.com/react-router/web/api/NavLink)

Reply:
done